openverse_catalog/dags/commoncrawl/sync_commoncrawl_workflow.py

```diff
@@ -33,52 +25,50 @@ def _empty_tsv_dir(tsv_directory):
 
 
-def create_dag():
-    dag = DAG(
-        dag_id=DAG_ID,
-        default_args=DAG_DEFAULT_ARGS,
-        start_date=datetime(2020, 1, 15),
-        schedule_interval="0 16 15 * *",
-        catchup=False,
-        tags=["commoncrawl"],
-    )
-
-    with dag:
-        create_dir_task = PythonOperator(
-            task_id="create_tsv_directory",
-            python_callable=os.makedirs,
-            op_args=[CRAWL_OUTPUT_DIR],
-            op_kwargs={"exist_ok": True},
-            depends_on_past=False,
-        )
-        sync_tsvs_task = BashOperator(
-            task_id="sync_commoncrawl_workflow",
-            bash_command=(
-                f"python {airflowHome}/dags/commoncrawl_s3_syncer/SyncImageProviders.py"
-            ),
-            env={
-                "S3_BUCKET": os.environ["S3_BUCKET"],
-                "OUTPUT_DIR": CRAWL_OUTPUT_DIR,
-                "AWS_ACCESS_KEY": os.environ["AWS_ACCESS_KEY"],
-                "AWS_SECRET_KEY": os.environ["AWS_SECRET_KEY"],
-            },
-        )
-        clean_tsvs_task = PythonOperator(
-            task_id="clean_commoncrawl_tsvs",
-            python_callable=clean_tsv_directory,
-            op_args=[CRAWL_OUTPUT_DIR],
-            depends_on_past=False,
-        )
-        empty_dir_task = PythonOperator(
-            task_id="empty_tsv_directory",
-            python_callable=_empty_tsv_dir,
-            op_args=[CRAWL_OUTPUT_DIR],
-            depends_on_past=False,
-        )
-
-        (create_dir_task >> sync_tsvs_task >> clean_tsvs_task >> empty_dir_task)
-
-    return dag
+dag = DAG(
+    dag_id=DAG_ID,
+    default_args={
+        **DAG_DEFAULT_ARGS,
+        "retries": 3,
+        "retry_delay": timedelta(days=1),
+        "execution_timeout": None,
+    },
+    schedule_interval="0 16 15 * *",
+    catchup=False,
+    tags=["commoncrawl"],
+)
 
+with dag:
+    create_dir_task = PythonOperator(
+        task_id="create_tsv_directory",
+        python_callable=os.makedirs,
+        op_args=[CRAWL_OUTPUT_DIR],
+        op_kwargs={"exist_ok": True},
+        depends_on_past=False,
+    )
+    sync_tsvs_task = BashOperator(
+        task_id="sync_commoncrawl_workflow",
+        bash_command=(
+            f"python {airflowHome}/dags/commoncrawl_s3_syncer/SyncImageProviders.py"
+        ),
+        env={
+            "S3_BUCKET": os.environ["S3_BUCKET"],
+            "OUTPUT_DIR": CRAWL_OUTPUT_DIR,
+            "AWS_ACCESS_KEY": os.environ["AWS_ACCESS_KEY"],
+            "AWS_SECRET_KEY": os.environ["AWS_SECRET_KEY"],
+        },
+    )
+    clean_tsvs_task = PythonOperator(
+        task_id="clean_commoncrawl_tsvs",
+        python_callable=clean_tsv_directory,
+        op_args=[CRAWL_OUTPUT_DIR],
+        depends_on_past=False,
+    )
+    empty_dir_task = PythonOperator(
+        task_id="empty_tsv_directory",
+        python_callable=_empty_tsv_dir,
+        op_args=[CRAWL_OUTPUT_DIR],
+        depends_on_past=False,
+    )
 
-globals()[DAG_ID] = create_dag()
```

Comment:
Is this globals modification no longer necessary?

Reply:
Yep! Airflow is already able to load [top-level DAGs](https://airflow.apache.org/docs/apache-airflow/stable/concepts/dags.html#loading-dags) into the DagBag. Now that we've pulled the DAG definiton out of the function, this isn't necessary 😄